communication/vehicleconnections/mavsdkstation.h

```diff
@@ -12,10 +12,15 @@
 #include <QMap>
 #include <QSharedPointer>
 #include <QSerialPortInfo>
+#include <QTimer>
+#include <QVector>
+#include <QPair>
 #include <mavsdk/mavsdk.h>
 #include <mavsdk/plugins/mavlink_passthrough/mavlink_passthrough.h>
 #include "mavsdkvehicleconnection.h"
 
+#define TIMEOUT 5
```

Comment:
the naming does not say what timeout this value refers to, would also say it is better to use a const instead of define:
`const int HEARTBEATTIMER_TIMEOUT_SECONDS = 5;`